css/style.css

```diff
@@ -0,0 +1,15 @@
+h2 {
+  font-family: Arial;
+  padding: 0 20px;
+  background: 	#add8e6;
+
+}
+.button {
+  background: #6495ed;
+}
+.main_list {
+  position: relative;
+  display: block;
+  margin-left: 50px;
+  padding: 10px;
+}
```

Comment:
Make sure that all files end with a new line.

Reply:
Deleted additional space on the fifth line.